frame/vamm/src/helpers/math.rs

```diff
@@ -0,0 +1,122 @@
+use crate::{Config, Error, Pallet};
+use frame_support::pallet_prelude::*;
+use sp_core::U256;
+use sp_runtime::{traits::Zero, ArithmeticError, FixedPointNumber};
+
+impl<T: Config> Pallet<T> {
```

Comment:
Could some of these functions be moved to composable-math so more pallets could utilize them?